### PR TITLE
Manual creation of kubevirtci provider for 1.29

### PR DESCRIPTION
**Pull request by dhiller**
Creates the 1.29 kubevirtci provider from a manual run of the automation.

### CODE REVIEW

**Comment by dhiller** [comment]
@brianmcarey @xpivarc looks good so far for a start. One conformance test failing: https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirtci/1093/check-provision-k8s-1.29/1724377867195781120

**Comment by dhiller** [comment]
/hold

waiting for https://github.com/kubevirt/project-infra/pull/3063 to get merged

**Comment by dhiller** [comment]
/test check-provision-k8s-1.29

Let's see whether this is reproducible.

**Comment by dhiller** [comment]
Conformance test failure seems to be reproducible: https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirtci/1093/check-provision-k8s-1.29/1724377867195781120

**Comment by dhiller** [comment]
@xpivarc

**Review by brianmcarey — COMMENTED** [review]
Changes look good to me. Maybe we should try to bump to the latest rc to see if that helps with the conformance test? 

/lgtm

File: cluster-provision/k8s/1.29/provision.sh [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+dnf install -y patch
+
+systemctl stop firewalld || :
+systemctl disable firewalld || :
+# Make sure the firewall is never enabled again
+# Enabling the firewall destroys the iptable rules
+yum -y remove firewalld
+
+# Required for iscsi demo to work.
+yum -y install iscsi-initiator-utils
```
Comment (oshoval) [inline cluster-provision/k8s/1.29/provision.sh:93]: https://github.com/kubevirt/kubevirtci/pull/1103
worth to do like this please

**Comment by dhiller** [comment]
/close 

Favor instead #1094

**Pull request closed without merging**